doc/specs/#3026 - Bypass Microsoft Store blocked for COM API.md

```diff
@@ -0,0 +1,65 @@
+---
+author: Roy MacLachlan RDMaclachlan/roy.maclachlan@microsoft.com
+created on: 2023-02-09
+last updated: 2023-02-09
+issue id: 3026
+---
+
+# Bypass Microsoft Store blocked for COM API
+
+"For [#3026](https://github.com/microsoft/winget-cli/issues/3026)"
```

Comment:
Wrong Issue :) The linked issue is Icons

Reply:
😅 Off my one number... It's supposed to be 3025.